src/app_service/service/ens/utils.nim

```diff
@@ -58,23 +59,12 @@ proc namehash*(ensName:string): string =
 
   result = "0x" & node.toHex()
 
-proc resolver*(usernameHash: string): string =
-  let payload = %* [{
-    "to": ENS_REGISTRY,
-    "from": "0x0000000000000000000000000000000000000000",
-    "data": fmt"{RESOLVER_SIGNATURE}{userNameHash}"
-  }, "latest"]
-
-  var resolverAddr = status_eth.doEthCall(payload).result.getStr()
-  resolverAddr.removePrefix("0x000000000000000000000000")
-  result = "0x" & resolverAddr
-
-proc contenthash(ensAddr: string): string =
+proc legacyContenthash(ensAddr: string): string =
```

Comment:
Is this being used? I think the get content hash was only used in order to resolve the resource url. You might not need this function at all

Reply:
Exactly, I think that the contentHash function was only use in order to build the resource URL. As we have a dedicated API now for resrouce URL I think this one is redundant but you are right, better to confirm with old master